lib/internal/inspector_network_tracking.js

```diff
@@ -22,18 +22,26 @@ function onClientRequestStart({ request }) {
     request: {
       url,
       method: request.method,
+      headers: request.getHeaders(),
```

Comment:
In CDP, the headers object is expected to be simple key-value string dictionary. However, the `request.getHeaders()` returns a dictionary with array value for duplicated keys. This will make the devtool abort for the following snippet:

```js
http.request('http://localhost:8888', {
  method: 'GET',
  headers: {
    'Cookie': ['foo=bar', 'bar=baz'],
  },
}, (res) => {
  //...
}).end();
```

Reply:
Addressed 01df4a8c13ac9e3b27b5f74bff70db58d342e3b4